code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm

```diff
@@ -181,6 +181,65 @@
 	robust_searching = 1
 	var/can_infest_dead = FALSE
 
+//Bloodman
+/mob/living/simple_animal/hostile/asteroid/hivelord/legion/bloodman
+	name = "bloodman"
+	desc = "You can still see what was once a human under the shifting mass of corruption."
+	icon = 'icons/mob/lavaland/lavaland_monsters.dmi'
+	faction = list("blooded")
+	icon_state = "bloodman"
+	icon_living = "bloodman"
+	icon_aggro = "bloodman"
+	icon_dead = "bloodman"
+	icon_gib = "syndicate_gib"
+	mob_biotypes = list(MOB_ORGANIC, MOB_HUMANOID)
+	mouse_opacity = MOUSE_OPACITY_ICON
+	obj_damage = 60
+	maxHealth = 30
+	health = 30 //dont want crew to have a hard time killing actual fodder
+	melee_damage_lower = 15
+	melee_damage_upper = 15
+	attacktext = "claws at"
+	speak_emote = list("echoes")
+	attack_sound = 'sound/weapons/pierce.ogg'
+	throw_message = "bounces harmlessly off of"
+	crusher_loot = /obj/item/crusher_trophy/legion_skull
+	loot = null
+	crusher_drop_mod = 10
+	brood_type = /mob/living/simple_animal/hostile/asteroid/hivelordbrood/bloodling
+	del_on_death = 1
+	stat_attack = DEAD
+	pass_flags = null
+	robust_searching = 1
+	var/mob/living/carbon/human/M
```

Comment:
What does this do
Assume i do not know how to search code

Reply:
fuck i copied those over from the original legion definitions and didn't realize I could just leave stuff out of a subtype if I wasn't gonna change it. the stat_attack = dead was SUPPOSED to make it so bloodmen wouldn't ignore corpses if there was one near them out of combat, but I just tried it again and I guess I was wrong since they aren't doing anything near a body with that stat_attack definition. Will be removing the lines I'm not changing